tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es3",
```

Comment:
Why this change?

Reply:
For wider ie support, for those who does not transpile `node_modules`

Reply:
I'm not sure we need to support ES3 builds. Do we? If somebody wants to downscale all the way down to ES3 it they should do it themselves. But if we do want to provide ES3 build, we would need to keep `/lib` folder and create separate `/es3` or `/dist/es3` folder.